RecoTracker/LSTCore/standalone/setup.sh

```diff
@@ -7,18 +7,23 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $DIR/code/rooutil/thisrooutil.sh
 
 ARCH=$(uname -m)
-if [[ $(hostname) == *lnx4555* ]]; then
-  export SCRAM_ARCH=el9_amd64_gcc12
-elif [[ $ARCH == "aarch64" || $ARCH == "arm64" ]]; then
-  export SCRAM_ARCH=el9_aarch64_gcc12
+if [ -z ${CMSSW_SEARCH_PATH+x} ]; then 
+  if [[ $(hostname) == *lnx4555* ]]; then
+    export SCRAM_ARCH=el9_amd64_gcc12
+  elif [[ $ARCH == "aarch64" || $ARCH == "arm64" ]]; then
+    export SCRAM_ARCH=el9_aarch64_gcc12
+  else
+    export SCRAM_ARCH=el8_amd64_gcc12
+  fi
+  export CMSSW_VERSION=CMSSW_14_1_0_pre3
+
+  source /cvmfs/cms.cern.ch/cmsset_default.sh
+  cd /cvmfs/cms.cern.ch/$SCRAM_ARCH/cms/cmssw/$CMSSW_VERSION/src
+  eval `scramv1 runtime -sh`
 else
-  export SCRAM_ARCH=el8_amd64_gcc12
+  cd $CMSSW_BASE/src
+  eval `scramv1 runtime -sh`
```

Comment:
isn't this redundant? (if `CMSSW_BASE` is set; the scram runtime was already called)

Reply:
Indeed, it is redundant, and I removed it. The `cd` could also be considered redundant but I kept it for convenience (since for the standalone-only setup we have to do a `cd -` and it is easier if we change directory also in the CMSSW case).